cypress-io/cypress
# fix: bring back initial loading spinner in launchpad

Closes: #24950

### User facing changelog


The launchpad will now show a loading spinner on startup to indicate that it has successfully started an is a loading state.

### Additional details


We had this previously but some changes to the launchpad in 10.11.0 unintentionally removed it. This PR adds the spinner in the `index.html` to display it as quickly as possible, before JS has even loaded and mounted the app.


### Steps to test


Open and Cypress project and make sure that you see a spinner before the Testing Type picker.

### How has the user experience changed?


Before - no spinner, blank screen until testing type cards appear:

![Screen Shot 2022-12-01 at 12 19 43 PM](https://user-images.githubusercontent.com/8340719/205118320-4947da1f-5c9e-4a94-b0b8-fcebece8cb4f.png)

After - spinner immediately on startup


![Screen Shot 2022-12-01 at 12 19 26 PM](https://user-images.githubusercontent.com/8340719/205118459-4025cb77-ed5f-4ba2-a648-8365c6d1230a.png)


### PR Tasks


- [na] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [na] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
